scipy/integrate/_cubature.py

```diff
@@ -172,6 +178,24 @@ def cubature(f, a, b, *, rule="gk21", rtol=1e-8, atol=0, max_subdivisions=10000,
     Genz-Malik is typically less accurate than Gauss-Kronrod but has much fewer nodes,
     so in this situation using "genz-malik" might be preferable.
 
+    Infinite limits are handeled with an appropriate variable transformation. Assuming
+    ``a = [a_1, ..., a_n]`` and ``b = [b_1, ..., b_n]``:
+
+    If :math:`a_i` and :math:`b_i` range over :math:`x \in (-\infty, \infty)`, the i-th
+    integration variable will use the transformation :math:`x = \frac{1-|t|}{t}` and
+    :math:`t \in (-1, 1)`.
+
+    If :math:`a_i` and :math:`b_i` range over :math:`x \in [a_i, \infty)`, the i-th
+    integration variable will use the transformation :math:`x = a_i + \frac{1-t}{t}` and
+    :math:`t \in (0, 1)`.
+
+    If :math:`a_i` and :math:`b_i` range over :math:`x \in (-\infty, b_i]`, the i-th
+    integration variable will use the transformation :math:`x = b_i - \frac{1+t}{t}` and
```

Comment:
I think $b_i - (1+t)/t$ is correct here, since the $(-\infty, b_i]$ case is handled by flipping the limits, negating $x$ and then applying the transformation for $[a_i, \infty)$.

Reply:
But if $0 < t < 1$ and $x = b_i - (1+t)/t$, then $x$ will range from $-\infty$ to $b_i-2$, and there is a “gap” $[b_i-2,b_i]$, right?

Reply:
Ah yes, thanks!